lib/src/main/java/io/ably/lib/rest/Auth.java

```diff
@@ -409,6 +403,21 @@ public JsonObject asJSON() {
 	 * @param options
 	 */
 	public TokenDetails authorize(TokenParams params, AuthOptions options) throws AblyException {
+		return authorize(params, options, true);
+	}
+
+	/**
+	 * Alias of authorize() (0.9 RSA10l)
+	 */
+	public TokenDetails authorise(TokenParams params, AuthOptions options) throws AblyException {
+		Log.w(TAG, "authorise() alias will be removed in 1.0");
+		return authorize(params, options);
+	}
+
+	/**
+	 * Internal authorize() where the third param chooses whether to force reauthorization.
+	 */
+	public TokenDetails authorize(TokenParams params, AuthOptions options, boolean force) throws AblyException {
```

Comment:
That sounds like a bit more of a restructure than I have done. My internal method is "authorize with a force flag", and thus leaves the logic pretty much as it was.